MOBILE_CLIENT/ANDROID/_MEDIUM/INSECURE_REGISTER_RECEIVER_FLAG/description.md

```diff
@@ -0,0 +1,6 @@
+The application calls the registerReceiver method with the argument flags set to RECEIVER_EXPORTED, which can be exploitable as it exposes the BroadcastReceiver to external applications, potentially leading to unauthorized access and security vulnerabilities.
+
+=== "Java"
```

Comment:
Maybe add a vulnerable manifest file snippet.

Reply:
the KB is related only to the specific issue of passing flag RECEIVER_EXPORTED to registerReceiver. It doesnt have anything to do with manifest file.